test/recipes/82-test_tfo_cli.t

```diff
@@ -0,0 +1,87 @@
+#! /usr/bin/env perl
+# Copyright 2022 The OpenSSL Project Authors. All Rights Reserved.
+#
+# Licensed under the Apache License 2.0 (the "License").  You may not use
+# this file except in compliance with the License.  You can obtain a copy
+# in the file LICENSE in the source distribution or at
+# https://www.openssl.org/source/license.html
+
+use strict;
+use warnings;
+
+use IPC::Open2;
+use OpenSSL::Test qw/:DEFAULT srctop_file bldtop_file/;
+use OpenSSL::Test::Utils;
+
+setup("test_tfo");
+
+plan skip_all => "test_tfo_cli needs sock enabled" if disabled("sock");
+plan skip_all => "test_tfo_cli needs tls < 1.3 enabled"
+    if disabled("tls1_0") && disabled("tls1_1") && disabled("tls1_2");
+plan skip_all => "test_tfo_cli needs tls enabled" if disabled("tls");
```

Comment:
It's quite confusing that this is necessary. If `disabled("tls")` is true then this implies that `disabled("tls1_0") && disabled("tls1_1") && disabled("tls1_2")` is also true. So this `if disabled("tls")` clause should be entirely redundant......

.....AH!! except the first skip is incorrect it should read:

`if disabled("tls1") && disabled("tls1_1") && disabled("tls1_2")`

Note that "tls1_0" is the incorrect name - it should be "tls1"

Reply:
Right because it was running when it wasn't supposed to!
One would think that `tls1_0` and `tls1` were aliases...Oh well.. quick patch.